components/patient-data/impl/pom.xml

```diff
@@ -121,6 +121,18 @@
       <groupId>javax.servlet</groupId>
       <artifactId>javax.servlet-api</artifactId>
     </dependency>
+    <dependency>
```

Comment:
This isn't needed, the `patient-data-api` module is already declared as a dependency above.